doc/api/buffer.md

```diff
@@ -3140,12 +3140,20 @@ added: v8.2.0
 #### `buffer.constants.MAX_LENGTH`
 <!-- YAML
 added: v8.2.0
+changes:
+  - version: v14.0.0
+    pr-url: https://github.com/nodejs/node/pull/32116
+    description: Value is changed from 2<sup>31</sup> - 1 to
+      2<sup>32</sup> - 1 on 64-bit architectures.
 -->
 
 * {integer} The largest size allowed for a single `Buffer` instance.
 
 On 32-bit architectures, this value currently is 2<sup>30</sup> - 1 (~1GB).
-On 64-bit architectures, this value currently is 2<sup>31</sup> - 1 (~2GB).
+
+On 64-bit architectures, this value currently is 2<sup>32</sup> (~4GB).
```

Comment:
it looks like the previous value was correct here?

Reply:
No, sorry. It's just the `- 1` that's missing here.

Reply:
Should I add the `  -1`?

Reply:
> Should I add the ` -1`?

Yes, the correct value is `2**32 - 1` on v14.x.

Reply:
Thank you! Fixed